src/main/java/games/strategy/triplea/util/AiBonusIncomeUtils.java

```diff
@@ -0,0 +1,38 @@
+package games.strategy.triplea.util;
```

Comment:
Yeah, I'd like to move 'util' type methods out of delegates to a different util package just not sure what exactly. Many of the delegate methods are potentially useful to the AI as well.